_pytest/config.py

```diff
@@ -80,6 +80,9 @@ def _prepareconfig(args=None, plugins=None):
     try:
         if plugins:
             for plugin in plugins:
+                if isinstance(plugin, str):
```

Comment:
Maybe warn in case of unicode strings as well? Not sure how pytest handles the 2/3 split though.